### PR TITLE
Fix 2476: Rename PropertyOf to AnyProperty

### LINKED ISSUE #2476: Rename PropertyOf to AnyProperty
Mirror other stdlib changes for Swift 2

### PR DESCRIPTION
**Pull request by AquaGeek**
Fixes #2476.

### CODE REVIEW

File: ReactiveCocoaTests/Swift/PropertySpec.swift [hunk 1]
```diff
@@ -108,16 +108,16 @@ class PropertySpec: QuickSpec {
 			}
 		}
 
-		describe("PropertyOf") {
+		describe("AnyProperty") {
 			describe("from a PropertyType") {
 				it("should pass through behaviors of the input property") {
 					let constantProperty = ConstantProperty(initialPropertyValue)
-					let propertyOf = PropertyOf(constantProperty)
+					let anyProperty = AnyProperty(constantProperty)
```
Comment (NachoSoto) [inline ReactiveCocoaTests/Swift/PropertySpec.swift]: Maybe this variable could just be `property`?

**Comment by NachoSoto** [comment]
This looks good!

**Comment by neilpa** [comment]
Yup. Can add an entry to the CHANGELOG as well?

**Comment by AquaGeek** [comment]
@NachoSoto I couldn't find any references to `PropertyOf` in the docs. Can you point me in the right direction?

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,10 @@ only affecting generic, custom signal/producer extensions.
 
 Maybe coming to a later alpha. See #2360.
 
+#### Renamed PropertyOf<T> to AnyProperty<T>
```
Comment (NachoSoto) [inline CHANGELOG.md]: Thanks!

**Comment by NachoSoto** [comment]
Awesome, thank you!

**Pull request merged**